### PR TITLE
Leveraging RC to cache data via rise-data-component

### PR DESCRIPTION
**Pull request by stulees**
- Updating dependency on `rise-google-sheet`
- Revised integration tests to mock the `getItem` call on rise-data within rise-google-sheet

### CODE REVIEW

**Comment by stulees** [comment]
@donnapep @rodrigopavezi Please review, thanks

File: gulpfile.js [hunk 1]
```diff
@@ -221,6 +222,10 @@
     });
   });
 
+  gulp.task("build-dev", function (cb) {
```
Comment (donnapep) [inline gulpfile.js]: @stulees Nice!

**Comment by stulees** [comment]
@donnapep @rodrigopavezi Please review last commit. e2e tests now pass locally and on CCI. Thanks

**Comment by donnapep** [comment]
👍

**Comment by stulees** [comment]
@rodrigopavezi Please review last commit, thanks

**Pull request merged**